airbyte-integrations/connectors/destination-clickhouse/src/main/kotlin/io/airbyte/integrations/destination/clickhouse/spec/ClickhouseConfiguration.kt

```diff
@@ -40,14 +42,16 @@ class ClickhouseConfigurationFactory :
             }
 
         return ClickhouseConfiguration(
-            pojo.hostname,
-            pojo.port,
-            protocol,
-            pojo.database,
-            pojo.username,
-            pojo.password,
-            pojo.enableJson ?: false,
-            pojo.getTunnelMethodValue(),
+            hostname = pojo.hostname,
+            port = pojo.port,
+            protocol = protocol,
+            database = pojo.database,
+            username = pojo.username,
+            password = pojo.password,
+            enableJson = pojo.enableJson ?: false,
+            tunnelConfig = pojo.getTunnelMethodValue(),
+            recordWindowSize = pojo.recordWindowSize,
+            bytesWindowSize = pojo.bytesWindowSize,
```

Comment:
I'm not sure about exposing bytes as that is tuned to fit within our connector memory, which people don't have control over in cloud